### PR TITLE
aurora-cli 0.16.0 update

### PR DESCRIPTION
**Pull request by thinker0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/aurora-cli.rb [hunk 1]
```diff
@@ -11,7 +11,8 @@ class AuroraCli < Formula
     sha256 "9b45a0ba82e22ff2e272c80977d047a16b5164edb0efc198ffc90ede170c5444" => :mavericks
   end
 
-  depends_on :java => "1.8+"
```
Comment (MikeMcQuaid) [inline Formula/aurora-cli.rb]: This no longer uses Java?

File: Formula/aurora-cli.rb [hunk 2]
```diff
@@ -11,7 +11,8 @@ class AuroraCli < Formula
...
+  patch :DATA
```
Comment (MikeMcQuaid) [inline Formula/aurora-cli.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Review by MikeMcQuaid — APPROVED** [review]

File: Formula/aurora-cli.rb (hunk 2 above)
Comment (MikeMcQuaid) [inline Formula/aurora-cli.rb]: This looks like it may be done without a patch.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Sorry, wrong change

File: Formula/aurora-cli.rb (hunk 1 above)
Comment (thinker0) [inline Formula/aurora-cli.rb]: Aurora CLI only Python

File: Formula/aurora-cli.rb (hunk 2 above)
Comment (thinker0) [inline Formula/aurora-cli.rb]: https://github.com/apache/aurora/commit/b272b8808c5d9832112bfd9b648e0ec8b9718414

Comment (thinker0) [inline Formula/aurora-cli.rb]: This means you must enter information.


```
thinker0-iMac:Formula thinker0$ brew reinstall aurora-cli
==> Reinstalling aurora-cli
==> Using the sandbox
==> Downloading https://www.apache.org/dyn/closer.cgi?path=/aurora/0.16.0/apache-aurora-0.16.0.tar.gz
==> Best Mirror http://mirror.apache-kr.org/aurora/0.16.0/apache-aurora-0.16.0.tar.gz
######################################################################## 100.0%
==> ./pants binary src/main/python/apache/aurora/kerberos:kaurora
Last 15 lines from /Users/thinker0/Library/Logs/Homebrew/aurora-cli/01.pants:
23:59:14 00:00     [parse]fatal: Not a git repository (or any of the parent directories): .git

               Executing tasks in goals: binary
23:59:15 00:01   [binary]
23:59:15 00:01     [binary-prep-command]
23:59:15 00:01     [python-binary-create]
23:59:15 00:01       [cache]
                   No cached artifacts for 1 target.
                   Invalidated 1 target.
23:59:20 00:06       [chroot]
FAILURE: Update --binaries-path-by-id to find binaries for Darwin x86_64 16.0.0.


23:59:20 00:06   [complete]
               FAILURE

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues

```

**Comment by thinker0** [comment]
@MikeMcQuaid double check sierre Mac OS

**Comment by MikeMcQuaid** [comment]
Does:
```
patch do
  url "https://github.com/apache/aurora/commit/b272b880.patch"
  sha256 ...
end
```

work instead? Additionally, please add a Ruby comment explaining why the patch is needed (not just the URL). Thanks!

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Comment by thinker0** [comment]
Thanks ^^

**Pull request closed without merging**